iOSClient/Data/NCDatabase.swift

```diff
@@ -415,6 +416,10 @@ class tableMetadata: Object, NCUserBaseUrl {
 
 extension tableMetadata {
     var fileExtension: String { (fileNameView as NSString).pathExtension }
+
+    var isPrintable: Bool {
+        classFile == NCCommunicationCommon.typeClassFile.image.rawValue || ["application/pdf", "com.adobe.pdf"].contains(contentType) || contentType.hasPrefix("text/")
+    }
```

Comment:
var fileExtension: String { (fileNameView as NSString).pathExtension } ?
Already exists the ext in  tableMetadata

Reply:
Yes, but that can be used if `ext` is not set or empty e.g. in the RenameFile. See also #1885